app/utils/identityHelpers.ts

```diff
@@ -4,8 +4,15 @@ import {
     ChosenAttributes,
     AttributeKey,
     AttributeKeyName,
+    PendingIdentity,
+    IdentityStatus,
+    ConfirmedIdentity,
+    RecoveredIdentity,
+    RejectedIdentity,
 } from './types';
 
+export const currentIdentityVersion = 1;
```

Comment:
We should document somewhere what the two versions mean. Something like:
```
// The identity was created using the legacy BLS12-381 key generation, decoding the seed
// as UTF-8.
identityVersion = 0

// The identity was created using the corrected BLS12-381 key generation, decoding the seed
// as hex.
identityVersion = 1
```
I am not sure if it should be here though.

Reply:
I have created a type `IdentityVersion`, and documented it there.